musysim.py

```diff
@@ -110,7 +175,27 @@ def mrand(self, e):
         sign = e // abs(e)
         return randint(1, abs(e)) * sign
 
+    def get_val(self, symbol):
```

Comment:
potentially refactor this away... have left it for now because it seems to do what it needs to without harming anything

Reply:
Do this later, not blocking anything now.